src/gc/unix/gcenv.unix.cpp

```diff
@@ -419,16 +412,15 @@ size_t GCToOSInterface::GetLargestOnDieCacheSize(bool trueSize)
 //  specify a 1 bit for a processor when the system affinity mask specifies a 0 bit for that processor.
 bool GCToOSInterface::GetCurrentProcessAffinityMask(uintptr_t* processMask, uintptr_t* systemMask)
 {
-    // TODO(segilles) processor detection
-    return false;
+    return !!::GetProcessAffinityMask(::GetCurrentProcess(), (PDWORD_PTR)processMask, (PDWORD_PTR)systemMask);
```

Comment:
I meant copy and paste the source here. I know it sounds strange, but I don't see a clean way that would work - the GC is being transformed to a completely standalone self-contained thing that can be built without any dependency on PAL.